code/modules/vending/modularpc.dm

```diff
@@ -19,7 +19,7 @@
 	premium = list(/obj/item/computer_hardware/card_slot = 2,
 		           /obj/item/computer_hardware/ai_slot = 2,
 		           /obj/item/computer_hardware/printer/mini = 2,
-		           /obj/item/computer_hardware/recharger/APC = 2,
+		           /obj/item/computer_hardware/recharger/apc = 2,
```

Comment:
if you wouldn't mind, give this a slightly longer name while you're at it. apc_recharger or something like that would be awesome.